lib/plottr_components/src/components/dashboard/account/proOnboarding/ProStep2.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'react-proptypes'
+import { t } from 'plottr_locales'
+import OnboardingStep from '../../../onboarding/OnboardingStep'
+import { StepBody, StepHeader } from '../../../onboarding/Step'
+import UnconnectedUploading from './Uploading'
+
+// is there a way to know that this step is done?
```

Comment:
Migration is a per-machine step, so we could use settings (or something else that's persisted to the HDD).

We can also go through the file list and find the files that aren't any cloud files, but that doesn't mean that the user skipped this step.

A combination of the two might be appropriate.
Track migrations using state persisted to the HDD and notify them that they could migrate more to the cloud if we find more files there.